packages/codemods/src/helpers/result.ts

```diff
@@ -0,0 +1,127 @@
+import { Chainable, Flattened } from './chainable';
+
+export interface Ok<R, E> extends ResultInternal<R, E> {
+  ok: true;
+  value: R;
+}
+
+export interface Err<R, E> extends ResultInternal<R, E> {
```

Comment:
sorry for the nit pick, but i think we should use full word `Error` here, and other places too

Reply:
I chose err because the Error constructor already exists and I don't want to overload it to much.

Reply:
Actually Error is not a constructor, it's used the exact same way that this error would be, `Error("Error is thrown")`